sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering-validation.models.ts

```diff
@@ -551,6 +576,49 @@ export class OfferingValidationModel {
     ),
   })
   yearOfStudy: number;
+  /**
+   * Indicates if the offering is an aviation offering.
+   */
+  @IsEnum(AviationYesNoOptions, {
+    message: getEnumFormatMessage(
+      userFriendlyNames.isAviationOffering,
+      AviationYesNoOptions,
+    ),
+  })
+  @ValidateIf((offering: OfferingValidationModel) => !!offering.programContext)
```

Comment:
Asking for my understanding, what will be the impact of not using the validate if here.